examples/splunk-hec-metrics-with-telemetrygen/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3"
+services:
+  telemetrygen:
+    image: ghcr.io/open-telemetry/opentelemetry-collector-contrib/telemetrygen:v0.71.0
```

Comment:
Is there any way to keep this at latest instead of a specific version?

Reply:
There is no guarantee that telemetrygen's latest docker image will continue to behave with the same arguments right now given how it is considered alpha, so I'd rather we keep a fixed version we know to work.

Reply:
Works for me, thanks for clarifying!